specs2/toolchain/BUILD

```diff
@@ -0,0 +1,42 @@
+load("@io_bazel_rules_scala//scala:providers.bzl", "declare_deps_provider")
+load("//scala/toolchains:toolchains.bzl", "declare_deps_toolchain")
+
+toolchain_type(
+    name = "specs2_toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "specs2_toolchain",
+    toolchain = ":specs2_toolchain_impl",
+    toolchain_type = ":specs2_toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+declare_deps_provider(
+    name = "specs2_deps_provider",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@io_bazel_rules_scala_org_specs2_specs2_common",
+        "@io_bazel_rules_scala_org_specs2_specs2_core",
+        "@io_bazel_rules_scala_org_specs2_specs2_fp",
+        "@io_bazel_rules_scala_org_specs2_specs2_matcher",
+    ],
+)
+
+declare_deps_provider(
+    name = "specs2_junit_deps_provider",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//external:io_bazel_rules_scala/dependency/specs2/specs2_junit",
```

Comment:
this is just leftovers, right? doesn't have to still be bind

Reply:
This PR does not remove binds, it just makes changes required to be able to remove them in the future. Simply removing binds will break some users. I don't think we want to force existing users to stop using binds right away, no? I think we should do binds removal as a separate PR.